handbook/communication/content_guidelines/style_and_mechanics.md

```diff
@@ -0,0 +1,986 @@
+# Style and mechanics
+
+We have an editorial style to help us keep our copy clear and consistent, and specific guidelines to write and structure content to make it easy for users to understand and act on.
+
+Remember, we strive for [effectiveness over correctness](../content_guidelines.md#effectiveness-over-correctness). Use these guidelines as a reference, not an authority, and always prioritize what's most effective for our target users.
+
+## Spelling
+
+Use the most popular US English spelling and phrasing.
+
+## Abbreviations & acronyms
+
+Avoid abbreviations, acronyms, and latinisms when possible. Use complete English words for clarity.
+
+<div class="usage">
+<div class="item yes">
+<h5>Yes</h5>
+<ul>
+<li>for example</li>
+<li>that is</li>
+<li>as opposed to</li>
+<li>through</li>
+</ul>
+</div>
+<div class="item no">
+<h5>No</h5>
+<ul>
+<li>e.g.</li>
+<li>i.e.</li>
+<li>v.</li>
+<li>via</li>
+</ul>
+</div>
+</div>
+
+**Further reading:**
+
+- "[Acronyms Seriously Suck](https://gist.github.com/klaaspieter/12cd68f54bb71a3940eae5cdd4ea1764)" from Elon Musk.
+
+## Adverbs and adjectives
+
+Try to avoid adjectives and adverbs. Using adjectives and adverbs (for example, it’s “easy” to get started with Sourcegraph) shapes perception and sets expectations, and can inadvertently lead to a negative emotional experience for our users. 
+
+Instead, look for opportunities to capture the same meaning through other forms. What makes an “easy” book easy relative to other books? Can that be used as a scale of relative difficulty?
+
+## Active voice vs. passive voice
+
+Use active voice and avoid passive voice. In active voice, the subject of the sentence does the action. In passive voice, the subject of the sentence has the action done to it.
+
+<div class="usage">
+<div class="item yes">
+<h5>Yes</h5>
+<ul>
+<li>Settings saved.</li>
```

Comment:
I think there are zombies here! 😵 

([We] "Saved settings" vs "Settings saved" [by Zombies]). Never realized how many save messages are passive; this is so common.

Reply:
Ha! This is a bad example for active voice.

```suggestion
<li>Added repository.</li>
```